cmd/image/sync/sync.go

```diff
@@ -29,18 +29,183 @@ import (
 
 // copy workload type for copy worker method
 type copyWorkload struct {
-	s3Cli     client.S3Client
+	s3Cli     SyncClient
 	srcBucket string
 	tgtBucket string
 	srcObject string
 }
 
+// instance item for source and target instances
+type InstanceItem struct {
+	Source SyncClient
+	Target []SyncClient
+}
+
 // sync constants
 const (
 	ServiceType     = "cloud-object-storage"
 	NoOfCopyWorkers = 20
```

Comment:
If they aren't used outside the package, then:

```suggestion
	serviceType     = "cloud-object-storage"
	noOfCopyWorkers = 20
```

Reply:
fixed